prow/jobs/jinja/config/label_sync.jinja2

```diff
@@ -114,6 +124,11 @@ default:
       name: area/security
       target: both
       addedBy: label
+    - color: 0052cc
+      description: Issues or PRs related to testing
+      name: area/testing
```

Comment:
seems pretty close to `area/infra` in actual usage.

Reply:
We have few issues that are related to testing without necessarily being related to prow or infrastructure. `area/infra` is a very generalistic one that don't necessarily include testing. I personally think label will be useful.